README.md

```diff
@@ -164,7 +163,10 @@ Being accepted in the whitelist as a part of the federation currently takes up t
 If you want to participate in the network by running a Raiden Service Bundle you need to register yourself as a provider. Currently, there are two conditions that must be fulfilled to run the RSB successfully.
 
 1. **Registering in the Services Registry On-Chain**  
-  - In order to register as a service provider you need to run the script [`register-service-provider.sh`](https://github.com/raiden-network/raiden-service-bundle/blob/master/register-service-provider.sh) as described in step 3 and 4 in [Installing the RSB](#installation/installing) written above. 
+  - In order to register as a service provider you need to run the script [`register-service-provider.sh`](https://github.com/raiden-network/raiden-service-bundle/blob/master/register-service-provider.sh).
+  - Make sure that the account is stored in `${DATA_DIR}/keystore/`. If not, the script will exit with an error and you cannot register as a service provider.
+  - Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service provider. Click [here](https://goerli.etherscan.io/address/0x735722704e365c37247bb3e4ec52d6c937c54539#readContract) to find out what the current price for a slot is. You will find the price under `3. currentPrice` denominated in RDN. Otherwise, the script will tell you the price, too.
```

Comment:
This mixes together goerli testnet and mainnet. It's okay for now, but we need to be aware that this should be updated for mainnet. Preferably we say SVT instead of RDN as long as we're on goerli.